specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/managedHsm.json

```diff
@@ -0,0 +1,1609 @@
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "A boolean value that indicates whether the name is available for you to use. If true, the name is available. If false, the name has already been taken or is invalid and cannot be used."
+        },
+        "reason": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The reason that a managed hsm name could not be used. The reason element is only returned if NameAvailable is false.",
+          "enum": [
+            "AccountNameInvalid",
+            "AlreadyExists"
+          ],
+          "x-ms-enum": {
+            "name": "Reason",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "An error message explaining the Reason value in more detail."
+        }
+      },
+      "description": "The CheckMhsmNameAvailability operation response."
+    },
+    "ManagedHSMSecurityDomainProperties": {
+      "type": "object",
+      "properties": {
+        "activationStatus": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Activation Status",
+          "enum": [
+            "Active",
+            "NotActivated",
+            "Unknown",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ActivationStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Active",
+                "description": "The managed HSM Pool is active."
+              },
+              {
+                "value": "NotActivated",
+                "description": "The managed HSM Pool is not yet activated."
+              },
+              {
+                "value": "Unknown",
+                "description": "An unknown error occurred while activating managed hsm."
+              },
+              {
+                "value": "Failed",
+                "description": "Failed to activate managed hsm."
+              }
+            ]
+          }
+        },
+        "activationStatusMessage": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Activation Status Message."
+        }
+      },
+      "description": "The security domain properties of the managed hsm."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    },
```

Comment:
You are highly recommended to remove your own definitions of these and use the definitions from common-specification/types/v3/types.json instead

Reply:
This is old existing file, not introduced in this PR, so skipping.